docs/streamtypes/tile-document/api.md

```diff
@@ -97,25 +95,26 @@ await doc.update({ foo: 'baz'}, { tags: ['baz'] })
 [:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_tile.tiledocument-1.html#update){:target="_blank"}
 
 
-### Parameters
+#### Parameters
 
-#### content
+##### content
 
 The new content of your document. This fully replaces any existing content in the document.
 
-#### metadata (optional)
+##### metadata (optional)
 
 Only fields that are provided in the metadata arg will be updated.  Fields not specified
 in the metadata arg will be left with their current value.
 
 [:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_stream_tile.tilemetadataargs-1.html){:target="_blank"}
 
 
-!!! info "Updating the `deterministic` parameter"
+!!! warning "Updating the `deterministic` parameter"
```

Comment:
this one makes sense to me to be a warning

Reply:
same as above